src/main/java/uk/gov/ons/ctp/integration/contactcentresvc/service/addressindex/model/PostcodeSearchResponseDTO.java

```diff
@@ -0,0 +1,20 @@
+package uk.gov.ons.ctp.integration.contactcentresvc.service.addressindex.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import java.util.ArrayList;
+import lombok.Data;
+
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
Is not that the default behaviour?

Reply:
No. You need the 'ignoreUnknown'. 
https://fasterxml.github.io/jackson-annotations/javadoc/2.6/com/fasterxml/jackson/annotation/JsonIgnoreProperties.html